lib/dfe/reference_data/itt/incentives.rb

```diff
@@ -451,20 +451,22 @@ module ITT
             scholarship: '17000',
             notes: 'geography 2020-2021'
           },
+          # rubocop:disable Lint/DuplicateHashKey
           '95eebd6c-69c5-4883-b3ff-c703b559bc4b' => {
             subject_name: 'classics',
             itt_categories_id: '95547d49-2a52-4ae0-b4ea-744e0da4bfa2',
             academic_cycle: ['2021'],
             bursary_amount: '26000',
             notes: 'classics and biology 2020-2021'
           },
-          '80464bd0-f509-4d54-b70a-add606b1844f' => {
+          '95eebd6c-69c5-4883-b3ff-c703b559bc4b' => {
```

Comment:
This introduces a duplicate Hash key. It's same key as line 455, which shouldn't be allowed. We need resolve this duplicate. Also we need to keep the DuplicateHashKey cop for this reason.